pkgs/applications/editors/vscode/extensions/contextmapper.context-mapper-vscode-extension/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, vscode-utils, substituteAll, graphviz, jdk11, makeWrapper }:
+vscode-utils.buildVscodeMarketplaceExtension rec {
+  mktplcRef = {
+    name = "context-mapper-vscode-extension";
+    publisher = "contextmapper";
+    version = "6.7.0";
+    sha256 = "sha256-vlDVqn1Je0eo5Nf2gyotSvhIa07tWCINe79RZSyMzcA=";
+  };
+
+  nativeBuildInputs = [
+    graphviz
+    makeWrapper
+  ];
+
+  postInstall = ''
+    wrapProgram $out/share/vscode/extensions/contextmapper.context-mapper-vscode-extension/lsp/bin/context-mapper-lsp \
+      --prefix JAVA_HOME : ${jdk11}
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Reasoning:
* https://github.com/atorres1985-contrib/nixpkgs/commit/451dfa6d84586b5cb83e6a7254689dff66b4523a